packages/story-editor/src/utils/getBlurHashFromImage.js

```diff
@@ -47,7 +43,11 @@ const getBlurHashFromImage = async (src) => {
 
   const trackTiming = getTimeTracker('load_get_blurhash');
   return new Promise((resolve, reject) => {
-    const worker = new Worker(); // lgtm[js/call-to-non-callable]
+    const worker = new Worker(
+      new URL('./generateBlurhash.worker.js', import.meta.url),
+      { type: 'module' }
```

Comment:
We don't output ESM, so this should not be a module worker. What issue did this solve / what issue are you trying to solve here?

Reply:
💭 I think this is all connected to the main problem: the `blurhash` code isn't being inlined in the worker script.

The `import` and `export` aren't getting transpiled. To take a closer look at this I changed the worker rule to
```js
{
    test: /\.worker\.js$/,
    type: 'asset/resource',
},
```

And webpack output for the worker script looks like:
```js
// packages/story-editor/src/utils/generateBlurhash.worker.js

// import { encode } from 'https://cdn.skypack.dev/blurhash@1.1.4';
import { encode } from 'blurhash';

const callback = function (event) {
  const {
    image,
    width,
    height,
    componentX,
    componentY
  } = event.data;
  console.log('✨✨✨ running the worker script');

  try {
    const blurHash = encode(image, width, height, componentX, componentY);
    postMessage({
      type: 'success',
      blurHash
    });
  } catch (error) {
    console.log('🚨🚨🚨 script error', error);
    postMessage({
      type: 'error',
      error
    });
  }
};

self.onmessage = callback; // Only needed for Rollup, not webpack v4.

export default {
  callback
};
```

So I think I needed to add `{ type: 'module' }` there because the webpack output for our worker script is using `import/export`

Reply:
💭 If we can figure out how to inline the `blurhash` code in this webpack output then I think we won't need to use `{ type: 'module' }`

Reply:
Worker-loader (although deprecated) can still work: [c416836](https://github.com/GoogleForCreators/web-stories-wp/pull/5792/commits/c416836d982f0395ea56425eee790186fce59f6b)

This can be a good fallback, but I think we should be able to do the same thing without using the `worker-loader` loader.